tests/cbmc/proofs/s2n_stuffer_free/s2n_stuffer_free_harness.c

```diff
@@ -21,13 +21,6 @@
 #include <cbmc_proof/cbmc_utils.h>
 
 void s2n_calculate_stacktrace() {}
```

Comment:
don't we have a library for this now

Reply:
We do as soon as we merge https://github.com/awslabs/s2n/pull/1926. I created issue https://github.com/awslabs/s2n/issues/2008 so we can clean up the harnesses on a separate PR.